database/migrations/sql/20190226_add-active-days.sql

```diff
@@ -0,0 +1,15 @@
+/*
+ * Add active days table for recording user interaction
```

Comment:
just noticed that the migration files use dashes instead of underscores, let's use underscores to be consistent.